main.go

```diff
@@ -26,6 +29,11 @@ var (
 )
 
 func main() {
+	app.PreAction(func(_ *kingpin.ParseContext) error {
+		debug.SetPrintCommands(*debugCommands)
+		debug.SetPackageGenerated(*debugPackageGenerated)
+		return nil
+	})
 	os.Exit(main1())
 }
```

Comment:
`main` should only call `main1` - so the extra stuff should go in the other func

Reply:
Done.